addonscript/api.py

```diff
@@ -0,0 +1,50 @@
+"""API helpers for addonscript."""
+
+from addonscript.utils import get_api_url, amo_put, amo_get, amo_download
+from addonscript.task import get_channel
+
+# https://addons-server.readthedocs.io/en/latest/topics/api/signing.html#uploading-a-version
+UPLOAD_VERSION = "api/v3/addons/{id}/versions/{version}/"
+
+# https://addons-server.readthedocs.io/en/latest/topics/api/signing.html#checking-the-status-of-your-upload
+UPLOAD_STATUS = "api/v3/addons/{id}/versions/{version}/upload/{upload_pk}/"
+
+
+async def do_upload(context, locale):
+    """Upload the language pack for `locale` to AMO.
+
+    Returns the JSON response from AMO
```

Comment:
It's good practice to do [Google-style docstrings](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html); I believe we may make that our python standard. Not a blocker, but it helps create api docs for the code.

This is probably more important for scriptworker than *scripts. Not a blocker.

Reply:
Wish pydocstyle supported napolean/google docstrings though, linting this stuff is always better than humans conforming.